packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -255,7 +260,7 @@ def test_ipv6(dcos_api_session, same_host):
         log.info('Purging application: {}'.format(origin_app.id))
         origin_app.purge(dcos_api_session)
         log.info('Purging application: {}'.format(proxy_app.id))
-        origin_app.purge(dcos_api_session)
+        proxy_app.purge(dcos_api_session)
```

Comment:
Shame on me 🤦‍♂️

Would you please elaborate on your commit message? Do we call a wrong marathon api for purge?

> Oddly enough `purge` removes all app and pods. I don't know why this
> would be a member method of a Marathon app.

https://github.com/jeschkies/dcos/blob/26c82e1d388529233fe20252f22b643da59acc9a/packages/dcos-integration-test/extra/test_networking.py#L99-L100
https://github.com/jeschkies/dcos/blob/26c82e1d388529233fe20252f22b643da59acc9a/packages/dcos-integration-test/extra/test_networking.py#L155-L156